lib/improver/tests/utilities/test_spatial.py

```diff
@@ -67,6 +67,12 @@ def test_basic_distance_to_grid_cells(self):
             self.cube, self.DISTANCE, self.MAX_DISTANCE_IN_GRID_CELLS)
         self.assertEqual(result, (3, 3))
 
+    def test_basic_no_limit(self):
+        """Test the distance in metres to grid cell conversion."""
```

Comment:
It would be good to update this, so it's clearer what this test is for. The point seems to be to test for when we use the default value of `max_distance_in_grid_cells`.